verifying-signatures.html

```diff
@@ -59,12 +84,29 @@
 gpg: Signature made Fri 17 Feb 2017 05:20:55 PM CET
 gpg:                using RSA key C1E4CBA3AD78D3AFD894F9E0B7A66F03B59076A8
 gpg: Good signature from "KeePassXC Release &lt;<span
-                class="eobfs">release 'AT` keepassxc ^DOT' org</span>&gt;" [unknown]</code></pre>
-
-    <p>In addition, there is a <em>*.DIGEST</em> file that contains the SHA-256 digest for a release file.
-        You can verify the <strong>integrity</strong> of a package by checking its hash digest with the following command:</p>
+                class="eobfs">release 'AT` keepassxc ^DOT' org</span>&gt;" [unknown]
+gpg: WARNING: This key is not certified with a trusted signature!
+gpg:          There is no indication that the signature belongs to the owner.
+Primary key fingerprint: BF5A 669F 2272 CF43 24C1  FDA8 CFB4 C216 6397 D0D2
+     Subkey fingerprint: C1E4 CBA3 AD78 D3AF D894  F9E0 B7A6 6F03 B590 76A8</code></pre>
+    <p>You want to see that "Good signature" line. It shows the .sig file must have been created from
+       the AppImage file by the key with fingerprint <code>C1E4CBA3AD78D3AFD894F9E0B7A66F03B59076A8</code>.
+    </p>
+    <p>The warning is there because in this example we have not taken the extra step of <em>trusting</em> that key.
+    </p>
+ 
+    <p>Remember the <em>*.DIGEST</em> file?
+        You can verify the <strong>integrity</strong> of the package you downloaded with the following command:</p>
 
     <pre><code>$ shasum -a 256 -c KeePassXC-$VERSION-x86_64.AppImage.DIGEST
 KeePassXC-$VERSION-x86_64.AppImage: OK</code></pre>
+    <p>The <code>shasum</code> program recalculates the SHA-256 hash digest of the package file
+       and compares it with the value in the <em>.DIGEST</em> file. If they match, this shows the package
+       was downloaded without errors and has not been changed since the <em>.DIGEST</em> was created.</p>
```

Comment:
You should add a note here that verifying digests is purely a transfer integrity check. It doesn't say anything about whether the download is authentic.

Reply:
I've shortened the text here to state it's just a transfer integrity check